pages/posts/stability.md

```diff
@@ -0,0 +1,137 @@
+standards track RFC.
+
+JSON Schema has always been at the "individual draft" stage of this process. It
+was never on a standards track. There have been two major reasons why JSON
+Schema hasn't progress further than this in the IETF process. The first reason
+is that JSON Schema keeps evolving. Every year we get feedback from our
+community and we make changes or add features to respond to that feedback. If
+JSON Schema were to be standardized, responding to our community would be a
+difficult and slow processes. We believe that continuing to respond to the needs
+of the community is crucial for JSON Schema and we want to be able to continue
+doing that.
+
+The second reason we haven't pursued a standards track RFC is that the
+circumstances JSON Schema is subject to is not really compatible with the normal
+I-D process. It would be hard to jump into a WG and have to conform to a process
+that doesn't really fit the realities of our situation.
+
+Normally, each time a draft is released, it replaces the previous draft. That
+makes sense when you are developing a spec that people don't rely on yet, but
+JSON Schema is widely used in almost every corner of the web. Every time we
+release a new draft, that draft will be used and depended on in production
+systems. It can't simply be considered obsolete. Implementations need to
+continue to support those older drafts including backward incompatible changes.
+
+So, when people hear "draft", they hear "unfinished" or "not production ready".
+That's not the way we treat our releases. Every release is expected and
+encouraged to be used in production. It's no different than OpenAPI releasing a
+new version. No one asks when OpenAPI is going to be "done". JSON Schema is just
+perceived differently because we call our releases "drafts".
+
+## The problem
+
+But this isn't just a branding problem. When people ask when JSON Schema is
+going to be out of "draft", what they really mean is, when is JSON Schema going
+to be "stable". They want to be able to write a schema and be sure that it will
+continue to work the same way no matter how JSON Schema evolves in the future.
+They want to be able to update their implementation and not have to update their
+schemas that were working just fine.
+
+This effects implementers as well. Needing to support multiple versions of JSON
+Schema with no backward or forward compatibility guarantees gets cumbersome and
+has led many implementations to drop support for older releases. When this
+happens, it can leave users with a choice to have to update all of their
+existing schemas that weren't broken or pin to a no-longer-supported version of
+the JSON Schema library they are using.
+
+## Our solution
+
+Those are the problems we're aiming to solve in the next release. Instead of
+continuing to release a new immutable and incompatible version of JSON Schema
+with each release, our next release will be a long lived version that is stable,
```

Comment:
I feel like long-lived should have a hyphen, but feel free to ignore that feeling.